### PR TITLE
added localstorage to tractors

### PR DESCRIPTION
**Pull request by NaveenRajaLaxmana**
Added localstorage to tractors part

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -47,6 +48,7 @@ const HeaderSimple = ({name}) =>{
 const App = () => {
   return (
     <DriverState>
+      <TractorState>
```
Comment (Dinesh1129) [inline App.js:51]: What if we want to access Driver inside tractor and vice versa?

File: screens/Tractors/AddEditTractor.js [hunk 2]
```diff
@@ -10,48 +13,116 @@ export const MyButton = ({value="Add/Edit",cb={}}) => {
...
+                setRegistrationNumber(current.registration)
+                setColor(current.color)
+           }
+           else{
+            setId(uuid.v4())
+           }
+   },[])
+   const OnSubmit = () => {
+    if(Name.trim() === "" || company.trim()=== "" || registrationNumber.trim()=== ""){
+        ToastAndroid.show("Please fill required fields",ToastAndroid.SHORT)
```
Comment (Dinesh1129) [inline screens/Tractors/AddEditTractor.js:46]: So if we want to handle for iOS, do we need to add check here?

File: App.js (hunk 1 above)
Comment (NaveenRajaLaxmana) [reply App.js:51]: Then,May be we can try to use one maincontext which contain all context as it value.I don't know much about it but we try Anna.

File: screens/Tractors/AddEditTractor.js (hunk 2 above)
Comment (NaveenRajaLaxmana) [reply screens/Tractors/AddEditTractor.js:46]: Yes, for iOS we use AlertIOS hook for the toast message here.

**Pull request merged**